style.css

```diff
@@ -14,34 +14,28 @@ html {
 }
 
 body {
-   padding: 0 4rem;
    font-size: 2rem;
-   background-color: #F5FFFA;
+   background-color: #fff;
    font-family: 'Open Sans', sans-serif;
-   /*-webkit-column-count: 2;
-   -webkit-column-gap: 4rem;
-   -webkit-column-rule-style: solid;
-   -webkit-column-rule-width: 1px;
-   -webkit-column-rule-color: #000;
-   -webkit-column-rule: 1px solid #000;*/
 }
 
-section #header {
-   background-color: #fff;
-   margin-bottom: 4rem 0;
+header .site-header {
```

Comment:
Initially I didn't but it wasn't responding and when I included header it fired off. I was also a bit puzzled. I'll take another look at it and let you know